packages/wallets/src/features/accounts/modules/ManualService/components/DocumentSelection/components/DocumentSelectionCard/DocumentSelectionCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { WalletText } from '../../../../../../../../components';
+import useDevice from '../../../../../../../../hooks/useDevice';
+import RightArrow from '../../../../../../../../public/images/navigation-chevron-right.svg';
```

Comment:
quill icons 🤔

Reply:
Hi @lubega-deriv, due to time constraints, I have a separate card for this.